test/uk/gov/hmrc/plasticpackagingtax/registration/views/amendment/group/ListGroupMembersPageSpec.scala

```diff
@@ -0,0 +1,159 @@
+
+  val registration: Registration = Registration("someID",
+                                                organisationDetails = OrganisationDetails(
+                                                  organisationType = Some(UK_COMPANY),
+                                                  incorporationDetails = Some(incorporationDetails)
+                                                ),
+                                                groupDetail = Some(
+                                                  groupDetails.copy(members =
+                                                    Seq(groupMember, groupMember)
+                                                  )
+                                                )
+  )
+
+  val form: Form[Boolean] = AddOrganisationForm.form()
+
+  "manage_group_members_page" must {
+    "have the back button" in {
+      val sut: HtmlFormat.Appendable = view(form, registration)(journeyRequest, messages)
+
+      sut.getElementById("back-link") must haveHref(realAppConfig.pptAccountUrl)
+    }
+
+    "contain title" in {
+      val sut: HtmlFormat.Appendable = view(form, registration)(journeyRequest, messages)
+
+      sut.select("title").text() must include(messages("amend.group.listMembers.title", 2))
+    }
+
+    "contain heading" in {
+      val sut: HtmlFormat.Appendable = view(form, registration)(journeyRequest, messages)
+
+      sut.select("h1").text() mustBe messages("amend.group.listMembers.title", 2)
+    }
+
+    "include the error summary" when {
+      "the form has errors" in {
+        val sut = view(form.withError("foo", "site.button.tryAgain"), registration)(journeyRequest,
+                                                                                    messages
+        )
+
+        sut.select("#error-summary-title").size() mustBe 1
+      }
+    }
+
+    "list the members" in {
+      when(mockListComponent.apply(any())(any())).thenReturn(Html("listed members component"))
+
+      val sut: HtmlFormat.Appendable = view(form, registration)(journeyRequest, messages)
+
+      val representativeMember =
+        ListMember(name = registration.organisationDetails.businessName.get,
+                   subHeading = Some(messages("amend.group.manage.representativeMember")),
+                   change = None
+        )
+      val listMember = ListMember(groupMember.businessName,
+                                  change = Some(
+                                    groupRoutes.ContactDetailsCheckAnswersController.displayPage(
+                                      groupMember.id
+                                    )
+                                  ),
+                                  remove = Some(
+                                    routes.ConfirmRemoveMemberController.displayPage(groupMember.id)
+                                  )
+      )
+
+      verify(mockListComponent).apply(refEq(Seq(representativeMember, listMember, listMember)))(
+        any()
```

Comment:
This style is somewhat at odds with existing tests which utilise "real" view components and verify page content rather than mocking page components and verifying invocation.

Reply:
trying to remove duplicate testing, Have changed to look for the presence of their css selectors.